src/v2/resources/clients.js

```diff
@@ -50,6 +50,36 @@ class Clients extends BaseResource {
     const URL = `${this.baseURL}/clients/${clientId}.json`;
     return this.makeRequest({ URL, method: 'delete' });
   }
+
+  /**
+   * This will update the client from the parameters passed.
+   * @param {object} Client contains the params to update the client.
+   *    [Required] clientId, project unique identificator.
```

Comment:
It should be client, right?
```suggestion
   *    [Required] clientId, client unique identificator.
```